packages/shared/src/localStorageBroadcastChannel.ts

```diff
@@ -1,16 +1,40 @@
+import { deprecated } from './deprecated';
+
 type Listener<T> = (e: MessageEvent<T>) => void;
 
 const KEY_PREFIX = '__lsbc__';
 
+/**
+ * @deprecated This class will be moved to @clerk/clerk-js in the next major version.
+ * It is only used internally by ClerkBroadcastChannel and should not be used directly.
+ */
```

Comment:
This is only used in `@clerk/clerk-js` so just deprecating it here and will colocate it in the next major